### PR TITLE
Wrap up fixes in types related to null checks in TS

### PR DESCRIPTION
**Pull request by sizov**
This PR finishes fixes of types related to enabling strict null checks.

@BenGladman I deleted you pre-commit hook because it looks like we don't need it anymore

### CODE REVIEW

File: src/client/src/apps/MainRoute/widgets/analytics/epics/epics.test.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 import { ReferenceActions, ConnectionActions } from 'rt-actions'
 import { analyticsServiceEpic } from './epics'
-import { ActionsObservable } from 'redux-observable'
+import { ActionsObservable, StateObservable } from 'redux-observable'
 import { AnalyticsActions } from '../actions'
 import { Action } from 'redux'
 import { of, Observable } from 'rxjs'
 import { map } from 'rxjs/operators'
 import { ServiceStubWithLoadBalancer } from 'rt-system'
+import { GlobalState } from '../../../../../StoreTypes';
```
Comment (sizov) [inline src/client/src/apps/MainRoute/widgets/analytics/epics/epics.test.ts]: :(

File: src/client/src/apps/MainRoute/data/referenceData/epics.test.ts [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import { MockScheduler } from 'rt-testing'
 import { referenceServiceEpic } from './epics'
 import { Action } from 'redux'
-import { ActionsObservable } from 'redux-observable'
+import { ActionsObservable, StateObservable } from 'redux-observable'
 import { of } from 'rxjs'
 import { ConnectionActions, ReferenceActions } from 'rt-actions'
 import { CurrencyPair, CurrencyPairMap } from 'rt-types'
+import { GlobalState } from '../../../../StoreTypes';
```
Comment (sizov) [inline src/client/src/apps/MainRoute/data/referenceData/epics.test.ts]: :(

File: src/client/src/apps/MainRoute/widgets/status-bar/logoTypes.ts [hunk 3]
```diff
@@ -1,9 +0,0 @@
-import * as OpenFinLogo from './images/openfinLogo.png'
```
Comment (sizov) [inline src/client/src/apps/MainRoute/widgets/status-bar/logoTypes.ts]: seems that it is not used

File: src/client/src/apps/MainRoute/widgets/analytics/epics/epics.test.ts (hunk 1 above)
Comment (rikoe) [reply src/client/src/apps/MainRoute/widgets/analytics/epics/epics.test.ts]: :( :(

Comment (bhavesh-desai-scratch) [reply src/client/src/apps/MainRoute/widgets/analytics/epics/epics.test.ts]: I thought BaseUrl is supported in CRA

Comment (sizov) [reply src/client/src/apps/MainRoute/widgets/analytics/epics/epics.test.ts]: `StoreTypes` is at the root of the project, not in any of the subfolders. I hope all this goes away when we switch to multiple packages in the future

**Pull request merged**